lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -50,19 +50,19 @@
     default: "/"
   cert_chain:
     description:
-      - The CA certificate chain in PEM encoded format.
-      - Note that prior to 2.4, this parameter expected a path to a file.
-        Since 2.4 this is now accomplished using a lookup plugin. See examples for detail.
+      - The path to the CA certificate chain in PEM encoded format.
+        As of 2.4 this can also be accomplished using a lookup plugin.
```

Comment:
I've used comments like this on the local copy I'm using.
Happy to change the formatting or add stuff like "see examples"

```
     - The path to, or content of the certificate body in PEM encoded format.
        As of 2.4 both path and content are accepted.  If the parameter is not a file, it is assumed to be content.
```

```
     - The path to, or content of the private key in PEM encoded format.
        As of 2.4 both path and content are accepted.  If the parameter is not a file, it is assumed to be content. 
```

```
     - The path to, or content of the CA certificate chain in PEM encoded format.
        As of 2.4 both path and content are accepted.  If the parameter is not a file, it is assumed to be content.
```